pkg/analyzer/nodes/node.go

```diff
@@ -66,7 +66,8 @@ func Analyze(nodeName, rootFSdir string, uncertifiedRHEL bool) (*Components, err
 	}
 	c := &Components{}
 	c.OSNamespace, c.OSComponents, c.CertifiedRHELComponents, _, err =
-		detection.DetectComponents(nodeName, files, nil, nil, uncertifiedRHEL)
+		detection.DetectComponents(nodeName, files, nil, nil,
+			detection.DetectComponentOpts{UncertifiedRHEL: uncertifiedRHEL, IsRHCOSRequired: false})
```

Comment:
`IsRHCOSRequired: false` is hardcoded here, but we want to have ability to set it to `true` when we call `Analyze` from Compliance, right?